### PR TITLE
Update README to write how to configure adapter

### PR DESCRIPTION
**Pull request by tricknotes**
For the new user, how to customize adapter is an important concern.
Additionally, README says:
> Check how to change the adapter in the sections below.

However how to change adapter was not documented anywhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,12 @@ Thanks!
 
 ## High-level behavior
 
+Choise an adapter from [adapters](lib/active_model/serializer/adapter):
```
Comment (NullVoxPopuli) [inline README.md]: `Choise` should be choose.

**Comment by NullVoxPopuli** [comment]
I'm surprised that wasn't already on that document. nice catch!

File: README.md (hunk 1 above)
Comment (tricknotes) [inline README.md]: Good catch @NullVoxPopuli !
I fixed it 🔧 https://github.com/rails-api/active_model_serializers/pull/1758/commits/90cfa92b5102dd11f7755b427b21318602279061

File: README.md [hunk 2]
```diff
@@ -96,6 +96,12 @@ Thanks!
 
 ## High-level behavior
 
+Choose an adapter from [adapters](lib/active_model/serializer/adapter):
```
Comment (beauby) [inline README.md]: Adapters have been migrated to `lib/active_model_serializers/adapter` IIRC.

Comment (tricknotes) [inline README.md]: Ouch! That's right.

**Comment by NullVoxPopuli** [comment]
LGTM, @tricknotes, thanks for the update!

**Pull request merged**